plugins/builds/index.js

```diff
@@ -949,9 +977,9 @@ async function handleDuplicatePipelines(config) {
 
     // Handle external events
     // If no join array and external and pipeline the same, should be same event
-    if (duplicatePipelineIds.length) {
+    if (pipelinesToStart.length) {
```

Comment:
should this be (pipelinesToStart.length > 0)

Reply:
I think both work; if `pipelinesToStart.length` evaluates as anything other than `0`, it will return `true`; otherwise will return `false`
I can add the check if it's more readable/clear that way

Reply:
Cool, this works